code/modules/mob/living/carbon/human/emote.dm

```diff
@@ -61,19 +63,25 @@
 			playsound(Y,'sound/effects/thunder.ogg', 90, 1)
 			spawn(10)
 				user.gib()
+		if(ishuman(user) || ismonkey(user))

```

Comment:
I highly doubt we will ever add cyborg farts or whatever but I'll cross that bridge when it comes